ci/report-exit.js

```diff
@@ -10,18 +17,28 @@ function main() {
         {
           name: 'Configuration',
           value: `${process.env.NODE_ENV}`,
+        },
+        { 
+          name: 'Logs',
+          value: `${logUrls}`,
         }
       ],
     },
   ]
-  const data = { embeds: message, username: 'jest-reporter' }
-  const out = child_process.execSync(
-    `curl -X POST \
-        -H "Content-Type: application/json" \
-        -d '${JSON.stringify(data)}' \
-        ${process.env.DISCORD_WEBHOOK_URL}`
-  )
-  console.log(out.toString())
+  const userName = 'jest-reporter'}
+  const data = { embeds: message, username: userName }
```

Comment:
In this case there is no real need to pull out `username` because it is only being used in one place

Reply:
Updated